src/wiki/plugins/whatlinkswhere/settings.py

```diff
@@ -0,0 +1 @@
+SLUG = "whatlinkswhere"
```

Comment:
Some suggestions for slug:

* `related`
* `related-articles`
* `article-network`
* `link-network`

etc?

Reply:
`link-network` sounds okay. `graph` sounds to graphical where a visualization is an option, but shouldn't be the default; and because it includes red links, it's really the network of `link`s, not of articles. `related` is just too generic.

Would you suggest that the whole plugin should then also be called `link_network`? That might reduce confusion.

Reply:
I think that `link_network` sounds great and can perhaps invite further use cases and solutions to be added to the plugin :+1:

Reply:
Perhaps using `link-network` (with a dash) for the URL slugs for the sake of convention.

Reply:
After thinking about it, I think I'll go for linknetwork to avoid any confusion whether it's a `-` or a `_`.